manifests/host.pp

```diff
@@ -1,10 +1,12 @@
 # == Define: dhcp::host
 #
+# @param parameters used for setting different parameters like e.g  next-server declarition, filename for the host
 define dhcp::host (
   Optional[Stdlib::IP::Address] $ip     = undef,
   Dhcp::Mac $mac,
   String $ddns_hostname                 = $name,
   Hash $options                         = {},
+  Hash $parameters                      = {},
```

Comment:
Would this be a better suggestion?

```suggestion
  Hash[String[1], Any] $parameters                      = {},
```

Rather than `Any` it could also be `String[1]` but I don't know if you also want to deal with `Integer`.